Chandra/cmd_states/update_cmd_states.py

```diff
@@ -395,15 +395,19 @@ def main():
 
     # Set val to indicate if the output h5file is the "flight" version.
     # In that case use ftp directory cmd_states, else cmd_states_test.
-    is_flight = opt.h5file == '/proj/sot/ska/data/cmd_states/cmd_states.h5'
+    is_flight = ((opt.h5file == '/proj/sot/ska/data/cmd_states/cmd_states.h5')
+                 or (opt.server == '/proj/sot/ska/data/cmd_states/cmd_states.db3'))
```

Comment:
This needs work, as if the h5 is testing but the server is flight we don't want to have the one process push to cmd_states instead of cmd_states flight.  This should just get checked for each of these two types of content.